mmp_to_musicxml/converter.py

```diff
@@ -203,16 +203,21 @@ class MMP_MusicXML_Converter:
 	
 	SPECIFIED_KEY_SIGNATURE = None
 	
-	def __init__(self, check_notes=False, key_signature=None):
+	def __init__ ( self, key_signature = None, params = None ):
 		logging.basicConfig(level=logging.DEBUG)
-		
-		if check_notes:
+
+		self.opts = params['opts']
```

Comment:
I think it might be good to have a ternary-like operation here in case params is None (and also without a check my tests will break lol)

maybe something like:
```
self.opts = params.get('opts', None) if params else None
self.minor = params.get('minor', False) if params else False
```